src/Sylius/Bundle/CoreBundle/Resources/config/test_services.xml

```diff
@@ -23,6 +23,7 @@
         <parameter key="sylius.random_invoice_number_generator.class">Sylius\Component\Core\Test\Services\RandomInvoiceNumberGenerator</parameter>
         <parameter key="sylius.behat.factory.default_france_channel.class">Sylius\Component\Core\Test\Services\DefaultFranceChannelFactory</parameter>
         <parameter key="sylius.behat.factory.default_channel.class">Sylius\Component\Core\Test\Services\DefaultChannelFactory</parameter>
+        <parameter key="sylius.behat.email_checker.class">Sylius\Component\Core\Test\Services\EmailChecker</parameter>
```

Comment:
I'd say let's keep it here and rename all "behat" services from here, because they are not only for Behat.